apps/backoffice-v2/src/lib/blocks/hooks/useDocumentBlocks/hooks/useCommentInput/useCommentInput.ts

```diff
@@ -0,0 +1,19 @@
+import { useCallback, useState } from 'react';
+
+export const useCommentInput = () => {
+  const [value, setValue] = useState<string | undefined>(undefined);
```

Comment:
Don't have to make a change but this would take the same.
```suggestion
  const [value, setValue] = useState<string>();
```